tests/support_server/setup.pm

```diff
@@ -425,6 +445,9 @@ sub run {
     setup_networks();
 
     if (exists $server_roles{pxe}) {
+        # PXE server cannot be configured on other ARCH than x86_64
+        # because 'syslinux' package only exists on it
+        die "PXE server is only supported on x86_64 architecture" unless check_var('ARCH', 'x86_64');
```

Comment:
Have a suspicion that this is SLE15 specific issue ?

Reply:
No it's not, as my support_server tests have been done with SLE12-SP3, it's an architecture issue.
Not try yet with SLE15, and as autoyast profile may be different this test would be done later during the beta stage.